README.md

```diff
@@ -0,0 +1,18 @@
+# conda-plugin-template
+
+This repository contains a mix of templates and tutorials related to conda plugins.
+
+
+## Tutorials
+
+To take a look at the tutorials, head over to [tutorials](./tutorials).
+
+
+## Templates
+
+We currently have a [cookiecutter](https://cookiecutter.io) template you can use to
+start a new conda plugin project. To use it, run the following command:
```

Comment:
I would add a note here saying that you need to have `cookiecutter` installed first; I know this is kinda obvious, but it helps when every step is spelled out for potential tutorial users